hw/dv/data/vcs/vcs.hjson

```diff
@@ -99,6 +99,17 @@
   vcs_cov_assert_hier:  ""
   vcs_cov_excl_files:   []
 
+  // pass and fail patterns
```

Comment:
Sorry, I meant keep the common patterns in `common_sim_cfg.hjson` and only add the tool specific ones here.

Reply:
Thanks for pointing it out. Just realize these variables can append more items in other hjson like run_opts.
Updated, PTAL